include/vrv/doc.h

```diff
@@ -72,8 +73,18 @@ class Doc : public Object {
     /**
      * Getter for the options
      */
+    ///@{
     Options *GetOptions() const { return m_options; }
-    void SetOptions(Options *options) { (*m_options) = *options; };
+    void SetOptions(Options *options) { (*m_options) = *options; }
+    ///@}
+
+    /**
+     * Getter for the resources
+     */
+    ///@{
+    const Resources &GetResources() const { return m_resources; }
+    Resources &ModifyResources() { return m_resources; }
```

Comment:
It would be clearer if renamed to `GetResourcesForModification` or `GetModifiableResources`  because the call to the methods does not actually modify the resources.